packages/server/src/restaurants/dishes/api/DishDto.ts

```diff
@@ -51,6 +113,12 @@ class UpdateDishDto extends CreateDishDto {}
 
 class FavouriteDishDto extends PickType(DishDto, ['id', 'name'] as const) {}
 
-class ShortenedDishDto extends OmitType(DishDto, ['allergens', 'nutritionalValue', 'ingredients'] as const) {}
+class ShortenedDishDto extends OmitType(DishDto, [
```

Comment:
To bardziej pytanie niż komentarz. Nie wiem czy dobrze zrozumiałam z tego PR https://github.com/CodersCamp2021-HK/CodersCamp2021.Project.Fullstack/pull/50, że `ShortenedDish` nie będzie nam potrzebne, ale mogę się mylić.

Reply:
Zmergowałam zmiany z maina i już tego nie ma :D